include/circt-c/Dialect/FIRRTL.h

```diff
@@ -17,8 +17,94 @@
 extern "C" {
 #endif
 
+/// Wraps a FIRRTL BundleType element.
+struct FirrtlBundleElement {
+  MlirIdentifier name;
+  bool isFlip;
+  MlirType type;
+};
+typedef struct FirrtlBundleElement FirrtlBundleElement;
+
+//===----------------------------------------------------------------------===//
+// Dialect API.
+//===----------------------------------------------------------------------===//
+
 MLIR_DECLARE_CAPI_DIALECT_REGISTRATION(FIRRTL, firrtl);
 
+//===----------------------------------------------------------------------===//
+// Type API.
+//===----------------------------------------------------------------------===//
+
+/// Return 'true' if this is any FIRRTL type.
+bool firrtlTypeIs(MlirType type);
```

Comment:
I'm not sure how I would name this, but `firrtlTypeIs` feels a little weird. The other `firrtlTypeIs...` predicates make sense. Maybe this could be something like `firrtlTypeIsFIRRTLType`... but that is also weird.

Reply:
Mmh, I was trying to decide which one sounded less chaotic, but at least `firrtlTypeIsFIRRTLType` is *regular*.